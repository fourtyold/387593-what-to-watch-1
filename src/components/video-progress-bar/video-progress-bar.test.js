import React from "react";
import renderer from "react-test-renderer";

import VideoProgressBar from "./video-progress-bar.jsx";

const options = {
  progress: 50,
  formattedTime: {
    hours: `01`,
    minutes: `30`,
    seconds: `00`
  }
};

it(`Video progress bar renders correctly`, () => {
  const tree = renderer.create(
      <VideoProgressBar
        progress={options.progress}
        formattedTime={options.formattedTime}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
