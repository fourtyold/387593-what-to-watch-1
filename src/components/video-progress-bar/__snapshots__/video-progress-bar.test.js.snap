// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video progress bar renders correctly 1`] = `
<div
  className="player__controls-row"
>
  <div
    className="player__time"
  >
    <progress
      className="player__progress"
      max="100"
      value={50}
    />
    <div
      className="player__toggler"
      style={
        Object {
          "left": "50%",
        }
      }
    >
      Toggler
    </div>
  </div>
  <div
    className="player__time-value"
  >
    01:30:00
  </div>
</div>
`;
